src/intermediate.rs

```diff
@@ -1310,6 +1310,7 @@ impl RustType {
                 }],
                 ConceptualRustType::Primitive(p) => p.cbor_types(),
                 ConceptualRustType::Rust(ident) => {
+                    eprintln!("{ident:?}");
```

Comment:
Did you mean to leave in these two `eprintln`?

Reply:
Removed